docs/doc/13-sql-reference/50-file-format-options.md

```diff
@@ -2,21 +2,26 @@
 title: Input & Output File Formats
 ---
 
-Databend accepts a variety of file formats both as a source and as a target for data loading or unloading. For example, you can load data into Databend from a file with the [COPY INTO table command](../14-sql-commands/10-dml/dml-copy-into-table.md) or the Streaming Load API. You can also unload data from Databend into a file with the [COPY INTO location command](../14-sql-commands/10-dml/dml-copy-into-location.md) command. To do so, you need to tell Databend what the file looks like using the following syntax:
+Databend accepts a variety of file formats both as a source and as a target for data loading or unloading. For example, you can load data into Databend from a file with the [COPY INTO table command](../14-sql-commands/10-dml/dml-copy-into-table.md) or the Streaming Load API. You can also unload data from Databend into a file with the [COPY INTO location command](../14-sql-commands/10-dml/dml-copy-into-location.md) command. 
+
+## Syntax
+
+To specify a file format in a statement, use the following syntax:
 
 ```sql
-FILE_FORMAT = ( TYPE = { CSV | TSV | NDJSON | PARQUET | XML } [ formatTypeOptions ] )
-```
+-- Specify a standard file format
+... FILE_FORMAT = ( TYPE = { CSV | TSV | NDJSON | PARQUET | XML } [ formatTypeOptions ] )
 
-`Type`: Specifies the file format. Must be one of the ones listed above that Databend supports.
+-- Specify a custom file format
+... FILE_FORMAT = ( FORMAT_NAME = '<your-custom-format>' )
+```
 
-:::note
-Databend currently supports XML as a source ONLY. Unloading data into an XML file is not supported yet.
-:::
+- Databend currently supports XML as a source ONLY. Unloading data into an XML file is not supported yet.
+- If FILE_FORMAT is not explicitly specified, the default format used is PARQUET.
```

Comment:
```suggestion

the rule of FILE_FORMAT to be used is as follows:

1. try use the FILE_FORMAT in select/copy if exists
2. try use the FILE_FORMAT in `create stage` if a stage is related. if FILE_FORMAT not specified when `create stage`, it is `PARQUET` by default.
3. use PARQUET
```

Reply:
If you don't specify the FILE_FORMAT when you COPY INTO or SELECT data from a stage, Databend will use the file format that you initially defined for the stage when you created it. In cases where you didn't explicitly specify a file format during the stage creation, Databend defaults to using the PARQUET format. If you specify a different FILE_FORMAT from the one you defined when creating the stage, Databend will prioritize the FILE_FORMAT specified during the operation.